xml/decoders/Piko_SmartDecoder_4.1_Sound.xml

```diff
@@ -11,14 +11,16 @@
 <!-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or  -->
 <!-- FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License  -->
 <!-- for more details.  													-->
-<decoder-config xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder-4-15-2.xsd">
+<decoder-config xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder.xsd">
```

Comment:
Why is `-4-15-2` removed from the `decoder-4-15-2.xsd` file name?

Reply:
Probably because this file was made from an older version.  The -4-15-2 was added recently.  Would be good to put it back.

Reply:
Does this entry now apply to all decoder files ??

<decoder-config xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder-4-15-2.xsd">

Reply:
Yes, I think so.

Reply:
These .xml decoder files have "decoder.xsd" instead "decoder-4-15-2.xsd"

[files with #decoder.xsd#.txt](https://github.com/JMRI/JMRI/files/14151922/files.with.decoder.xsd.txt)

Reply:
Well, I will change the files I have modified.

  C:\Users\Uli\Documents\GitHub\JMRI_Uli\xml\decoders\piko\cv257.512extendedmap_bank1-4.xml
  C:\Users\Uli\Documents\GitHub\JMRI_Uli\xml\decoders\piko\PanePikoExtendedFmap_v4.1.xml
  C:\Users\Uli\Documents\GitHub\JMRI_Uli\xml\decoders\Piko_SmartDecoder_4.1_Sound.xml
  C:\Users\Uli\Documents\GitHub\JMRI_Uli\xml\decoders\uhlenbrock\cv257.512extendedmap_bank1-4.xml
  C:\Users\Uli\Documents\GitHub\JMRI_Uli\xml\decoders\uhlenbrock\PaneExtendedFmap.xml

Reply:
Changes completed

Reply:
> These .xml decoder files have "decoder.xsd" instead "decoder-4-15-2.xsd"

For example https://github.com/JMRI/JMRI/blob/master/xml/decoders/piko/PanePikoExtendedFmap_v4.1.xml has `decoder-4-15-2.xsd`. It was changed in PR #12813.

Reply:
> Changes completed

Thank you!